ansible_base/authentication/models/authenticator_user.py

```diff
@@ -42,6 +42,7 @@ class AuthenticatorUser(AbstractUserSocialAuth, AbstractCommonModel):
         on_delete=models.CASCADE,
         help_text=_("The local DB user related to this authenticator user."),
     )
+    email = models.EmailField(default=None, null=True, blank=True, help_text=_("The e-mail associated with this authenticator user."), max_length=250)
```

Comment:
I was initially concerned with the 250 length here but I think its likely fine.

Reply:
I can check if we can remove this, I had initially added this, as I had initially used a CharField, and max_length was required for that field

Reply:
I can remove this, by default an emailfield has a max length of 254 - 
https://docs.djangoproject.com/en/5.2/ref/models/fields/#emailfield